README.md

```diff
@@ -9,3 +9,410 @@ https://github.com/nginxinc/docker-nginx
 
 # Disclaimer
 This is not an official Google product.
+
+# <a name="about"></a>About
+
+This image contains an installation Nginx 1.x.
+
+For more information, see the
+[Official Image Marketplace Page](https://console.cloud.google.com/marketplace/details/google/nginx1).
+
+Pull command (first install [gcloud](https://cloud.google.com/sdk/downloads)):
+
+```shell
+gcloud auth configure-docker && docker -- pull marketplace.gcr.io/google/nginx1
+```
+
+Dockerfile for this image can be found [here](https://github.com/GoogleCloudPlatform/nginx-docker/tree/master/1).
+
+# <a name="table-of-contents"></a>Table of Contents
+* [Using Kubernetes](#using-kubernetes)
+  * [Running Nginx](#running-nginx-kubernetes)
+    * [Start a Nginx web server](#start-a-nginx-web-server-kubernetes)
+    * [Use a persistent data volume](#use-a-persistent-data-volume-kubernetes)
+  * [Web server configuration](#configuration-kubernetes)
+    * [Viewing existing configuration](#viewing-existing-configuration-kubernetes)
+    * [Using configuration volume](#using-configuration-volume-kubernetes)
+    * [Moving the web content to Nginx](#move-web-content-kubernetes)
+  * [Testing the web server](#testing-the-web-server-kubernetes)
+    * [Accessing the web server from within the container](#accessing-the-web-server-from-within-the-container-kubernetes)
+* [Using Docker](#using-docker)
+  * [Running Nginx](#running-nginx-docker)
+    * [Start a Nginx web server](#start-a-nginx-web-server-docker)
+    * [Use a persistent data volume](#use-a-persistent-data-volume-docker)
+  * [Web server configuration](#configuration-docker)
+    * [Viewing existing configuration](#viewing-existing-configuration-docker)
+    * [Using configuration volume](#using-configuration-volume-docker)
+    * [Moving the web content to Nginx](#move-web-content-docker)
+  * [Testing the web server](#testing-the-web-server-docker)
+    * [Accessing the web server from within the container](#accessing-the-web-server-from-within-the-container-docker)
+* [References](#references)
+  * [Ports](#references-ports)
+  * [Volumes](#references-volumes)
+
+# <a name="using-kubernetes"></a>Using Kubernetes
+
+Consult [Launcher container documentation](https://cloud.google.com/launcher/docs/launcher-container)
```

Comment:
This link is getting 404. Can you please change it by: https://cloud.google.com/marketplace/docs/container-images

Reply:
Fixed